common/config/.golangci.yml

```diff
@@ -256,8 +256,8 @@ issues:
       linters:
         - errcheck
         - maligned
-    - path: _test\.go$
-      text: "dot-imports: should not use dot imports"
```

Comment:
This change will be overridden, this file is auto generated

Reply:
Where's the source then? We made a similar change a few weeks ago.

Reply:
Fixed.